tinder_api/facebook_auth_token.py

```diff
@@ -22,16 +22,19 @@ def get_facebook_access_token(email, password):
     :param password: Facebook password.
     '''
     s = robobrowser.RoboBrowser(user_agent=MOBILE_USER_AGENT, parser='lxml')
+    s.session.cookies.update(COOKIE)
     s.open(FACEBOOK_AUTH)
+
     f = s.get_form()
     f['pass'] = password
     f['email'] = email
     s.submit_form(f)
     f = s.get_form()
+
     try:
-        s.submit_form(f, submit=f.submit_fields['__CONFIRM__'])
+        s.submit_form(f, submit=f.submit_fields['__CONFIRM__'], allow_redirects=False)
         access_token = re.search(
-            r"access_token=([\w\d]+)", s.response.content.decode()).groups()[0]
+            r"access_token=([\w\d]+)", s.response.headers['Location']).groups()[0]
```

Comment:
The new `/confirm` endpoint returns a redirect in the header, but that also contains the access_token so we just ignore (fatal) redirects and grab the token from a new location!